ui/page/creatorMemberships/supporterArea/index.js

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
+import MembershipsPage from './view';
+import { selectActiveChannelClaim } from 'redux/selectors/app';
+import { selectAccountChargesEnabled } from 'redux/selectors/stripe';
+import { doTipAccountStatus } from 'redux/actions/stripe';
+import { selectMyChannelClaims } from 'redux/selectors/claims';
+import { userHasMembershipTiers } from 'redux/selectors/memberships';
+import { doListAllMyMembershipTiers } from 'redux/actions/memberships';
+
+const select = (state, props) => ({
+  hasTiers: userHasMembershipTiers(state),
+  bankAccountConfirmed: selectAccountChargesEnabled(state),
+  activeChannelClaim: selectActiveChannelClaim(state),
+  myChannelClaims: selectMyChannelClaims(state),
+});
+
+const perform = {
+  doTipAccountStatus,
+  doListAllMyMembershipTiers,
+};
+
+export default connect(select, perform)(MembershipsPage);
```

Comment:
<img width="826" alt="image" src="https://user-images.githubusercontent.com/64950861/191065018-8d8db64b-67da-431b-8d65-e4e6b624e6f3.png">

:small_blue_diamond: Would be nice if can clean up, or we can discuss alternative like using a generic variable like `view` to avoid this recurring issue.